sympy/functions/combinatorial/numbers.py

```diff
@@ -1923,6 +1933,13 @@ def nT(n, k=None):
     >>> nT('1'*4)
     5
 
+    Another way of stating the above is that the number of partitions
+    of `n` identical items into `k` or fewer parts is just the number
+    of partititions of `n + k` identical items into `k` parts,
```

Comment:
This comment (minus "Another way...") could also be used to justify the `2*k >= n` test. Then we have `k >= n - k`, so *all* partitions of `n - k` have `k` or fewer parts.

Reply:
I changed it. Are there any more changes that i need to make.

Reply:
@smichr  @jksuom  is there anything more to change in this pr.